.github/workflows/gateway-conformance.yml

```diff
@@ -40,14 +40,15 @@ jobs:
 
       # 4. Run the gateway-conformance tests
       - name: Run gateway-conformance tests
-        uses: ipfs/gateway-conformance/.github/actions/test@v0.0
+        uses: ipfs/gateway-conformance/.github/actions/test@v0.1
         with:
           gateway-url: http://127.0.0.1:8040
           json: output.json
           xml: output.xml
           html: output.html
           markdown: output.md
           subdomain-url: http://example.net
+          specs: -dnslink-resolver,-ipns-resolver
```

Comment:
Explicitly disable these specs; hopefully, we haven't missed any tests.

When we know how to load the dnslink and ipns record into the car gateway, we can reenable these.